pkg/chrootuser/user.go

```diff
@@ -0,0 +1,127 @@
+package chrootuser
```

Comment:
Is this the latest version from buildah?

Reply:
It is now,